lib/turbolinks.rb

```diff
@@ -30,16 +31,11 @@ def referer
 
         require 'action_dispatch/routing/redirection'
         ActionDispatch::Routing::Redirect.class_eval do
-          def call_with_turbolinks(env)
-            status, headers, body = call_without_turbolinks(env)
-
-            if env['rack.session'] && env['HTTP_X_XHR_REFERER']
-              env['rack.session'][:_turbolinks_redirect_to] = headers['Location']
-            end
-
-            [status, headers, body]
+          if defined?(prepend) && Rails.version >= '4'
```

Comment:
I was lazy and did the same as in https://github.com/rails/turbolinks/pull/496/files :P
Don't think we need it. Let's see.